hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/model/HoodieFlinkRecord.java

```diff
@@ -193,14 +194,14 @@ public Object getColumnValueAsJava(Schema recordSchema, String column, Propertie
   private Object getColumnValueAsJava(Schema recordSchema, String column, Properties props, boolean allowsNull) {
     boolean utcTimezone = Boolean.parseBoolean(props.getProperty(
         HoodieStorageConfig.WRITE_UTC_TIMEZONE.key(), HoodieStorageConfig.WRITE_UTC_TIMEZONE.defaultValue().toString()));
-    RowDataQueryContext rowDataQueryContext = RowDataAvroQueryContexts.fromAvroSchema(recordSchema, utcTimezone);
+    RowDataQueryContext rowDataQueryContext = RowDataQueryContexts.fromSchema(HoodieSchema.fromAvroSchema(recordSchema), utcTimezone);
     return rowDataQueryContext.getFieldQueryContext(column).getValAsJava(data, allowsNull);
   }
 
   private Object getColumnValue(Schema recordSchema, String column, Properties props) {
```

Comment:
Can we just change the param to take in `HoodieSchema` so that then in line 204 we dont have to do `fromAvroSchema`, or does this result in large changes?